site/docs/guide.md

```diff
@@ -157,6 +157,14 @@ workspace.
   <td><code>//foo/...:all-targets</code></td>
   <td>All targets (rules and files) in all packages beneath the directory <code>foo</code>.</td>
 </tr>
+<tr>
+  <td><code>//...</code></td>
+  <td>All rules in all non-<a href="external.html">external</a> packages.</td>
```

Comment:
While you're here, I think `rules` in the other rows above should really mean `rule targets`. Do you mind changing those as well?

Reply:
Good idea. Done.